Makefile

```diff
@@ -60,16 +61,20 @@ cluster/prepare:
 cluster/clean:
 	make uninstall
 	-kubectl delete -f deploy/role.yaml
+	-kubectl delete -f deploy/clusterrole.yaml
 	-kubectl delete -n $(NAMESPACE) -f deploy/role_binding.yaml
 	-kubectl delete -n $(NAMESPACE) -f deploy/service_account.yaml
 	-kubectl delete -n $(NAMESPACE) -f deploy/crds/mdc_v1alpha1_mobiledeveloperconsole_crd.yaml
 	-kubectl delete -n $(NAMESPACE) -f deploy/mdc_v1alpha1_mobileclient_crd.yaml
 	-kubectl delete namespace $(NAMESPACE)
 
 
-.PHONY: install
+.PHONY: install-operator
```

Comment:
See https://github.com/aerogear/mobile-developer-console-operator/pull/12#discussion_r300636173 to see why I separated operator installation and MDC provisioning.